### ISSUE TITLE
`--ignore` not ignoring file in conjunction with testmon

### ISSUE THREAD

**Issue by jacksongoode** [tarpas/pytest-testmon#187]
We've recently a strange result from the call `pytest --ignore=test_dir/ignored_tests.py --testmon` where it actually runs all the tests from `ignored_test.py`. The initial pytest pre-log shows the deselected tests but then collects all (110) tests. I can confirm this happens even when the testmondata was removed. This is after a library upgrade.

```
platform linux -- Python 3.8.10, pytest-7.1.2, pluggy-1.0.0
testmon: libraries upgrade, changed files: 17, skipping collection of 29 files, environment: default
rootdir: ******, configfile: pytest.ini
plugins: testmon-1.3.4, anyio-3.6.1
collected 110 items
```

Should I be making this call with `--testmon-forceselect`?

**Comment by janbernloehr** [comment]
I think this was introduced in 1.34 since it was not present in 1.33.

**Comment by jacksongoode** [comment]
Since then, I resolved this by reverting this command back to `pytest -k 'not ignored_tests.py' --testmon-forceselect` for others that might need an alternative. However, in this case, the folder I want ignored is still scanned.

**Comment by tarpas** [comment]
released as 1.3.5dev4, install with pip install "pytest-testmon==1.3.5dev4" please
